app/components/version-list/template.html

```diff
@@ -16,16 +16,16 @@
         <a href='{{ versions.path }}/v{{version.versionNumber}}/' class="govuk-link govuk-link--no-visited-state govuk-!-font-weight-bold">Version {{ version.versionNumber }}</a>
       </td>
       <td class="govuk-table__cell">
-        {% if version.status === 'in design' %}
+        {% if version.status == 'in design' %}
```

Comment:
Is this intentional?
Is strict comparison not required here?

Reply:
Because this is nunjucks, rather than JavaScript, the behaviour is different. It was previously never matching, which I think is because nunjucks is checking whether it's the exact same instance, rather than the same string contents.

Reply:
I see, thanks for the explanation.